doc/Advanced.md

```diff
@@ -1212,6 +1212,11 @@ or like this respectively:
 {% endif %}
 ```
 
+### Test and debug 
+
+To see how a config file will look like once templated, simply open a browser at this address: `{{apricot_endpoint}}:32188/components/qc/ANY/any/tpc-pulser-calib-qcmn?process=true`
+Replace `{{apricot_endpoint}}` by the value you can find in Consul under `o2/runtime/aliecs/vars/apricot_endpoint` (it is different on staging and prod). 
+
```

Comment:
probably everyone will realise, but what you will get by replacing `apricot_endpoint` will be:
```
myhost.cern.ch:32188:32188/components/qc/ANY/any/tpc-pulser-calib-qcmn?process=true
```
notice the double port